src/flexbox/nasa.html

```diff
@@ -0,0 +1,52 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>NASA page</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./nasa.css">
+</head>
+<body>
+    <div class="main-container">
+        <div class="nasa">
+            <h2>NASA</h2>
+        </div>
+        <footer>
```

Comment:
remove the `<footer>` div. it's useless. or remove the `<div class="social">` up to you. but fix the css to match.